### ISSUE TITLE
Using a plane geometry for images 360 (or annotation type element or an image)

### ISSUE THREAD

**Issue by worlderland** [potree/PotreeDesktop#36]
Hello Potree community,

I am currently banging my head against getting the images 360 icons (the white spheres) to look like the attached picture.
![images 360 icons](https://user-images.githubusercontent.com/42766911/154426619-b04e9766-65c6-4614-9abb-40df2276a700.jpg)

After some research I found a possible solution, plane geometry, however I'm founding it extremely hard to implement it the way I'd like (also to have the plane geometries to look at the camera the same way the annotations do).

In conclusion I was wondering whether anyone else has come across this issue and whether they have resolved it.

Thank you in advance.
